src/Tempest/Discovery/DiscoveryDiscovery.php

```diff
@@ -42,7 +42,9 @@ public function storeCache(): void
 
     public function restoreCache(Container $container): void
     {
-        $discoveryClasses = unserialize(file_get_contents(self::CACHE_PATH));
+        $discoveryClasses = unserialize(file_get_contents(self::CACHE_PATH), [
```

Comment:
because the payload can be dynamic we can't really provide a list here. We'll allow all classes